webapp/components/wpt-flags.js

```diff
@@ -40,20 +40,15 @@ Object.defineProperty(wpt, 'ClientSideFeatures', {
       'colorHomepage',
       'diffFromAPI',
       'displayMetadata',
-      'fetchManifestForTestList',
       'githubCommitLinks',
       'githubLogin',
       'interopScoreColumn',
       'permalinks',
       'processorTab',
       'queryBuilder',
       'queryBuilderSHA',
-      'reftestAnalyzerMockScreenshots',
-      'reftestIframes',
       'searchCacheInterop',
       'showBSF',
-      'showTestType',
```

Comment:
This is true on [staging](https://pantheon.corp.google.com/datastore/entities;kind=Flag;ns=__$DEFAULT$__/query/kind?project=wptdashboard-staging)

Reply:
so is `showTestRefURL`

Reply:
Yes, however I'm fairly sure they have zero effect - the actual usage of them relies on `this.manifest` being filled, but the top-level flag being off means thats always null. I'd love if you could help me double-check that though!

Reply:
Yep good call! `fetchManifestForTestList` is never used -> `this.manifest` is always null -> `testType` is always ''. I set a few break points at the removed code in this PR and they are never triggered.